server.js

```diff
@@ -7715,6 +7715,53 @@ camp.route(/^\/nsp\/npm\/(?:@([^/]+)?\/)?([^/]+)?(?:\/([^/]+)?)?\.(svg|png|gif|j
   }
 }));
 
+// bundle size for npm packages
+camp.route(/^\/bundlephobia\/(min|gzip)\/(?:@([^/]+)?\/)?([^/]+)?(?:\/([^/]+)?)?\.(svg|png|gif|jpg|json)?$/,
+  cache((data, match, sendBadge, request) => {
+  // A: /bundlephobia/(min|gzip)/:package.:format
+  // B: /bundlephobia/(min|gzip)/:package/:version.:format
+  // C: /bundlephobia/(min|gzip)/@:scope/:package.:format
+  // D: /bundlephobia/(min|gzip)/@:scope/:package/:version.:format
+  const capturedResultType = match[1];
+  const capturedScopeWithoutAtSign = match[2];
```

Comment:
Much as I love verbose names, I think you could drop `captured` from these, and shorten this one to `scope`.

Reply:
Cool.